kernelci/api/helper.py

```diff
@@ -116,10 +116,13 @@ def create_job_node(self, job_config, input_node,
                 'kernel_revision': input_node['data']['kernel_revision'],
             },
         }
-        # for test kind, if node.data.kernel_type - copy to test node
-        # it is required for LAVA templates
-        if 'kernel_type' in input_node['data'] and job_config.kind == 'test':
-            job_node['data']['kernel_type'] = input_node['data']['kernel_type']
+        # Test-specific fields inherited from parent node (kbuild or
+        # test) if available
+        if job_config.kind == 'test':
+            job_node['data']['kernel_type'] = input_node['data'].get('kernel_type')  # noqa
```

Comment:
We have a lot of `#noqa` all over the place, often because of line length checks... Shall we consider adding a config for pycodestyle with an increased max line length? (I mean, it's 2024, we have screens and editors that can fit more than 79 chars per line)

Reply:
Absolutely, I wasn't expecting such changes to land in this PR, I created #2396 so we can further discuss this.